BinUtils/BinUtils/main.swift

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
I wasn't completely sure what to do with this file. Should they be converted to tests?